package/boot/uboot-ath79/patches/400-ath79-add-support-for-NEC-AR9344-Aterm-series.patch

```diff
@@ -0,0 +1,316 @@
+From f7b37647177f30f13586bc09a85ab2fe7249ae17 Mon Sep 17 00:00:00 2001
+From: INAGAKI Hiroshi <musashino.open@gmail.com>
+Date: Wed, 24 Apr 2024 23:54:46 +0900
+Subject: [PATCH] ath79: add support for NEC AR9344 Aterm series
+
+---
+ arch/mips/dts/Makefile                |  1 +
+ arch/mips/dts/nec,ar9344-aterm.dts    | 35 +++++++++++++++
+ arch/mips/mach-ath79/Kconfig          |  5 +++
+ board/nec/ar9344_aterm/Kconfig        | 30 +++++++++++++
+ board/nec/ar9344_aterm/Makefile       |  3 ++
+ board/nec/ar9344_aterm/ar9344_aterm.c | 60 +++++++++++++++++++++++++
+ configs/nec_ar9344_aterm_defconfig    | 64 +++++++++++++++++++++++++++
+ include/configs/nec_ar9344_aterm.h    | 28 ++++++++++++
+ 8 files changed, 226 insertions(+)
+ create mode 100644 arch/mips/dts/nec,ar9344-aterm.dts
+ create mode 100644 board/nec/ar9344_aterm/Kconfig
+ create mode 100644 board/nec/ar9344_aterm/Makefile
+ create mode 100644 board/nec/ar9344_aterm/ar9344_aterm.c
+ create mode 100644 configs/nec_ar9344_aterm_defconfig
+ create mode 100644 include/configs/nec_ar9344_aterm.h
+
+diff --git a/arch/mips/dts/Makefile b/arch/mips/dts/Makefile
+index 14fbce597b..6734fde805 100644
+--- a/arch/mips/dts/Makefile
```

Comment:
Please refresh this patch by `make package/uboot-ath79/{clean,refresh} V=s`. Not sure if this PR can be accepted, but I am personally happy to see the support of ath79 u-boot.

Reply:
Thanks, I'll do...